youtube_dl/extractor/extractors.py

```diff
@@ -646,7 +646,7 @@
 )
 from .podomatic import PodomaticIE
 from .pokemon import PokemonIE
-from .polskieradio import PolskieRadioIE
+from .polskieradio import PolskieRadioIE, PolskieRadioProgrammeIE
```

Comment:
That does not seem to be the standing style in that file. Parentheses are only used when the imports are broken down into separate lines, and that's only done when the line would otherwise be too long.